chrome/entrypoint.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+if [ "$(ls -A /home/chrome/.fonts/)" ]; then
+  fc-cache -f -v
+fi
+
+(ulimit -n 65000 || true) && (ulimit -p 65000 || true) && google-chrome-stable \
+  --enable-automation \
+  --disable-background-networking \
+  --disable-background-timer-throttling \
+  --disable-backgrounding-occluded-windows \
+  --disable-renderer-backgrounding \
+  --disable-breakpad \
+  --disable-client-side-phishing-detection \
+  --disable-default-apps \
+  --disable-dev-shm-usage \
+  --disable-extensions \
+  --disable-gpu \
+  --disable-popup-blocking \
+  --disable-prompt-on-repost \
+  --disable-sync \
+  --disable-translate \
```

Comment:
You should add a certain number of additional options:
1. `--disable-hang-monitor` to disable hang popup
2. `--noerrdialogs` to suppress all error dialogs
3. `--deny-permission-prompts` to prevent permission prompts
4. `--disable-component-extensions-with-background-pages` to disable extensions with background pages

```suggestion
  --disable-component-extensions-with-background-pages \
  --deny-permission-prompts \
  --disable-hang-monitor \
  --noerrdialogs \
  --disable-translate \
```

Reply:
I added them to the chrome. Do we need them in others images too?

Reply:
Why don't, let's add to every image